bot/cogs/snakes.py

```diff
@@ -29,6 +35,29 @@ def __init__(self, bot: AutoShardedBot):
         :return: A dict containing information on a snake
         """
 
+    async def get_snek_qwant_json(self, snake_name):
+        """
+        Gets the json from Unsplash for a given snake query
+        :param snake_name: name of the snake
+        :return: the full JSON from the search API
+        """
+        url = "https://api.unsplash.com/search/photos?client_id" \
+              "={0}&query={1}+snake".format(os.environ.get("UNSPLASH_CLIENT_ID"), snake_name)
```

Comment:
this string is a bit messy. prefer implicit string joins inside paranthesis and f-strings to \ line continuation and formats.

```py
client_id = os.environ.get("UNSPLASH_CLIENT_ID")
url = (
    "https://api.unsplash.com/search/photos?client_id" 
    f"={client_id}&query={snake_name}+snake"
)